CONTRIBUTING-GUIDE.md

```diff
@@ -0,0 +1,32 @@
+
+## Keploy Ecosystem: A Gateway to Contributing
+
+| Web Presence                                  | Description                                                                                                                                                                                                                                                                                                                                              | GitHub Repository                                                       | Contribution Guide                                                                                 | Issue Tracker                                                         | Latest Release                                                       |
+| --------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------- | -------------------------------------------------------------------- |
+| [Keploy.io](https://keploy.io/)               | Keploy, a developer-centric backend testing tool, offers features like combined test coverage, EBPF instrumentation, CI/CD integration, and multi-purpose mocks.                                                                                                                                                                                         | [keploy/keploy](https://github.com/keploy/keploy)                       | [Contribute Here](https://github.com/keploy/keploy/blob/main/CONTRIBUTING.md)                      | [View Issues](https://github.com/keploy/keploy/issues)                | [Release Info](https://github.com/keploy/keploy/releases)            |
+| _Not Available_                               | Client SDK for Go. Generates realistic mocks/stubs for applications, enhancing unit tests with minimal manual effort.                                                                                                                                                                                                                                    | [keploy/go-sdk](https://github.com/keploy/go-sdk)                       | [Contribute Here](https://github.com/keploy/go-sdk/blob/main/CONTRIBUTING.md)                      | _Not Available_                                                       | [Release Info](https://github.com/keploy/go-sdk/releases)            |
```

Comment:
we have web presence of our sdk's - 
Go SDK - https://pkg.go.dev/github.com/keploy/go-sdk/v2